tests/bearlib/aspects/TasteTest.py

```diff
@@ -17,6 +17,7 @@ def test__init__defaults(self):
         assert taste.description is ''
         assert taste.suggested_values is ()
         assert taste.default is None
+        assert taste.languages in [(), ('C#',)]
```

Comment:
@adhikasp Look at this https://github.com/coala/coala/blob/master/coalib/bearlib/aspects/taste.py#L75